Code/Legacy/CryCommon/platform_impl.cpp

```diff
@@ -238,22 +237,6 @@ void InitRootDir(char szExeFileName[], uint nExeSize, char szExeRootName[], uint
     }
 }
 
-//////////////////////////////////////////////////////////////////////////
-bool CrySetFileAttributes(const char* lpFileName, uint32 dwFileAttributes)
```

Comment:
you forgot to delete the header definition in platform.h#339

Reply:
Good catch